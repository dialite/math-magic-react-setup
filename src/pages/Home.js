import '../modules-css/home.css';

const Home = () => (
  <div className="main-cont">
    <div className="home-cont">
      <h3 className="welcome">Welcome to our Page!</h3>
      <p className="paragraph-one">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint dolor cupidatat
        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint dolor cupidatat
        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur.
      </p>
      <p className="paragragh-two">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint dolor cupidatat
        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint dolor cupidatat
        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur.Duis aute irure reprehenderit
        in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
      </p>
    </div>
  </div>
);

export default Home;
