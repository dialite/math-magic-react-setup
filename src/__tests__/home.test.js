import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('render Home', () => {
  const renderHome = renderer.create(<Home />).toJSON();
  expect(renderHome).toMatchSnapshot();
});
