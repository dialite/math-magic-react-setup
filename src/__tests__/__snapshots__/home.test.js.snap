// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home 1`] = `
<div
  className="HomeContainer"
>
  <h1
    className="marginLeftAndRight"
  >
    Welcome To Math Magician
  </h1>
  <div
    className="marginLeftAndRight"
  >
    <p>
      Mathematics is the science that deals with the logic of shape, quantity and arrangement. Math is all around us, in everything we do. It is the building block for everything in our daily lives, including mobile devices, computers, software, architecture (ancient and modern), art, money, engineering and even sports.
    </p>
    <p>
      Since the beginning of recorded history, mathematical discovery has been at the forefront of every civilized society, and math has been used by even the most primitive and earliest cultures. The need for math arose because of the increasingly complex demands from societies around the world, which required more advanced mathematical solutions, as outlined by mathematician Raymond L. Wilder in his book "Evolution of Mathematical Concepts(opens in new tab) " (Dover Publications, 2013).
    </p>
    <p>
      The more complex a society, the more complex the mathematical needs. Primitive tribes needed little more than the ability to count, but also used math to calculate the position of the sun and the physics of hunting. "All the records — anthropological and historical — show that counting and, ultimately, numeral systems as a device for counting form the inception of the mathematical element in all cultures," Wilder wrote in 1968.
    </p>
  </div>
</div>
`;
